servers/quarkus-tests/build.gradle.kts

```diff
@@ -43,6 +43,8 @@ dependencies {
   implementation(libs.testcontainers.mongodb)
   implementation(libs.docker.java.api)
 
+  implementation(libs.cassandra.quarkus.tests)
```

Comment:
Maybe exclude the testcontainers-cassandra dependency from this one and add 
```
implementation(libs.testcontainers.cassandra)
```
to avoid potential version conflicts.

Reply:
In this case I think we should use the cassandra-quarkus BOM artifact and also exclude the driver, wdyt?

Reply:
Quarkus-BOM's already in there as an enforcedPlatform dependency.